core-bundle/src/Security/Authentication/ContaoLoginAuthenticationListener.php

```diff
@@ -31,24 +31,25 @@
 use Symfony\Component\Security\Http\Session\SessionAuthenticationStrategyInterface;
 use Symfony\Contracts\EventDispatcher\EventDispatcherInterface;
 
+/**
+ * @internal
+ */
```

Comment:
This is not needed, because event listeners are explicitly excluded from our BC promise: https://docs.contao.org/dev/internals/bc-promise/

Reply:
👍 I'll remove it.

Reply:
Fixed in 9ae780b55.